src/plugins/vis_augmenter/public/utils/utils.ts

```diff
@@ -50,6 +55,13 @@ export const getAugmentVisSavedObjs = async (
   visId: string | undefined,
   loader: SavedAugmentVisLoader | undefined
 ): Promise<ISavedAugmentVis[]> => {
+  const config = getUISettings();
+  const isAugmentationEnabled = config.get(PLUGIN_AUGMENTATION_ENABLE_SETTING);
+  if (!isAugmentationEnabled) {
+    throw new Error(
+      'Visualization augmentation is disabled, please enable visualization:enablePluginAugmentation.'
```

Comment:
Out of scope and unrelated to feature anywhere - but ideally we'd make it easy for these types of configuration errors to directly deep link to the relevant setting. I'll open an issue or add to our existing issue about error handling.

Reply:
added to https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1625#issuecomment-1561499646